specs/vep-1739-multiple-python-versions/diagrams/high_level_sequence_plantUML_source.txt

```diff
@@ -0,0 +1,18 @@
+@startuml
+actor DataEngineer as engineer
+participant ControlService as cs
+participant K8sNamespace_DataJobBuilders as namespace
+database Database as db
+database Registry as ecr
+
+engineer -> cs : vdk create
+cs -> db : Register new data job
+cs -> engineer : Download sample data job
```

Comment:
The arrow direction indicates that the sender is the Control Service and is asking the user to download sample data job. 

Be careful to use the correct notation (https://www.lucidchart.com/pages/uml-sequence-diagram) 

I would remove it though. "Download sample data job" is not an important step for this proposal.

Reply:
Removed.